### DISCUSSION TITLE
Alpine not initializing / Carry is null error

### DISCUSSION THREAD

**Discussion by jdarkins** [livewire/livewire#7590] [1. Help]
I am trying to upgrade from v2 -> v3 but I've run into an issue where on some pages Alpine doesnt seem to be triggering the initialized event. 

Both component blades have the same script block & extend the same layout file which includes `<livewire:scripts />`, and when I view the page source all of these scripts are included - examples below.

Both console logs are triggered on the working page, however the `alpine initialized` log is never shown on the non working page. I also have a `Uncaught TypeError: carry is null` console error on the page. 

```
@push('scripts')
    <script>
        console.log('script loaded')
        document.addEventListener('alpine:initialized', function () {
            console.log('alpine initialized')
            Alpine.store('pagination').nextPageHooks.push(async function () {
                const res = await @this.continue()

                if (res && res.success) {
                    return true
                }
            })
        });
    </script>
@endpush
```
```
<!-- Livewire Scripts -->
<script src="/livewire/livewire.js?id=f477dd12"
```

**Comment by abdallahalsamman** [comment]
Same here

**Reply by abdallahalsamman** [reply]
Found solution here

https://github.com/livewire/livewire/discussions/6301#discussioncomment-6829066

> you can't bind to Eloquent Models in V3 unless you set legacy_model_binding to true. But we don't recommend using model binding, instead you should use FormObjects.
